src/App.vue

```diff
@@ -18,9 +18,12 @@ export default {
   name: "MainComponents",
   data() {
     return {
-      username: localStorage.getItem("token"),
+      username: localStorage.getItem("loginToken") || "",
     };
   },
+  updated() {
+    this.username = localStorage.getItem("loginToken");
```

Comment:
@JatinThakur14 Why is `username` = `loginToken` - makes no sense?

Reply:
Actually I don't have a login token yet so I was using username as loginToken. When the user logs in or sign up his username will be set as loginToken and he will be able use map